README.md

```diff
@@ -24,44 +24,33 @@ The React SDK is compatible with `React 16.3.0 +`
 import {
   createInstance,
   OptimizelyProvider,
-  OptimizelyExperiment,

```

Comment:
^^ not in scope of your changes but you might want to update in line 15:
'library of react components'
to 'library of react components and hooks to...'